Bowling/Bowling/Score.cs

```diff
@@ -0,0 +1,109 @@
+﻿using NUnit.Framework.Constraints;
+using NUnit.Framework.Internal.Execution;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+
+namespace BowlingApp
+{
+    public class Score
+    {
+        public const int frameNumberTotal = 10;
+        public const int frameNumberWithBonusRoll = 10;
+
+        public List<Frame> CreateScoreCard(string bowlingMarks)
+        {
+            var placeCount = 0;
+            List<Frame> framesCollection = new List<Frame>();
+            FrameNoBonusRollFactory frameNoBonusRoll;
```

Comment:
There is no need for this uninitialized variable to exist up here.  Further down you can just say:

`FrameNoBonusRollFactory  frameNoBonusRoll = new FrameNoBonusRollFactory();`